tests/test_network_models.py

```
@@ -0,0 +1,212 @@
+        VLANConfiguration(**{**VALID_VLAN_CONFIG, "vlan_id": 0})
+    with pytest.raises(ValidationError):
+        VLANConfiguration(**{**VALID_VLAN_CONFIG, "vlan_id": 4095})
+
+    # Test port validation
+    with pytest.raises(ValidationError):
+        VLANConfiguration(
+            **{
+                **VALID_VLAN_CONFIG,
+                "tagged_ports": [1, 2],
+                "untagged_ports": [2, 3],
+            }
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)